templates/zerver/find_account.html

```diff
@@ -43,7 +46,7 @@ <h1 class="get-started">{{ _("Find your Zulip accounts") }}</h1>
                     {% endtrans %}
                 </p>
                 <form class="form-inline" id="find_account" name="email_form"
-                  action="{{ current_url() }}" method="post">
+                  action="{{ current_url }}" method="post">
```

Comment:
This seems like a bug -- isn't `current_url` a function? Please test carefully.

Reply:
This shouldn't be a bug after the changes made in registration.py as the url is a string not a function

Reply:
I guess I must have seen a stale version, OK.